runtime/src/accounts.rs

```diff
@@ -180,11 +180,13 @@ impl Accounts {
             // If a fee can pay for execution then the program will be scheduled
             let mut called_accounts: Vec<Account> = vec![];
             for key in &message.account_keys {
-                called_accounts.push(
-                    AccountsDB::load(storage, ancestors, accounts_index, key)
-                        .map(|(account, _)| account)
-                        .unwrap_or_default(),
-                );
+                if !message.program_ids().contains(&key) {
```

Comment:
Probably being dense but didn't the `program_ids` member of message go by the wayside?

Reply:
The field did go away, but the `program_ids()` method is still pretty useful.